assets/js/blocks-registry/payment-methods/extensions-config.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { CanMakePayment } from '@woocommerce/type-defs/payments';
+
+type CanMakePaymentExtensionCallbacks = Record< string, CanMakePayment >;
+export type ExtensionNamespace = keyof CanMakePaymentExtensionsCallbacks;
+export type PaymentMethodName = keyof CanMakePaymentExtensionCallbacks;
+
+export type CanMakePaymentExtensionsCallbacks = Record<
+	PaymentMethodName,
+	CanMakePaymentExtensionCallbacks
+>;
+
+// Keeps callbacks registered by extensions for different payment methods
+/* eslint prefer-const: 0 */
```

Comment:
Is this the right way to disable it?

Reply:
As far as I know yes. Do you have anything else in mind?

Reply:
I thought this would disable that rule for the rest of the file, we don't have anything else affected by it, but we _could_ in future, so let's just disable it for the next line specifically.

`// eslint-disable-next-line prefer-const`

Reply:
You are right, I did not notice I wasn't just disabling the next line ✋